src/reanimated2/js-reanimated/index.ts

```diff
@@ -24,7 +24,12 @@ if (isWeb()) {
 
 const reanimatedJS = new JSReanimated();
 
-global._makeShareableClone = (c) => c;
+global._makeShareableClone = () => {
+  throw new Error(
```

Comment:
I'm a little worried about this change because there is a possible path to call this method on the Web:

`runOnJS -> makeShareableCloneOnUIRecursive -> _makeShareableClone``

Reply:
You have this piece of code there:

```TS
export function makeShareableCloneOnUIRecursive<T>(value: T): ShareableRef<T> {
  'worklet';
  if (USE_STUB_IMPLEMENTATION) {
    // @ts-ignore web is an interesting place where we don't run a secondary VM on the UI thread
    // see more details in the comment where USE_STUB_IMPLEMENTATION is defined.
    return value;
  }
  ```
so _makeShareableClone won't be called on web

Reply:
Yup, `USE_STUB_IMPLEMENTATION` (which is equal to `shouldBeUseWeb()` 😄) prevents `_makeShareableClone` from being called.